summernote/summernote
# Hide air popover in code view

#### What does this PR do?

- feature: back to air mode from code view
- fix: hide air popover when code view activated

#### Where should the reviewer start?

in the src/js/base/module
- Codeveiw.js
- AirPopover.js

#### How should this be manually tested?

- drag contents in air mode, then toolbar is pop over
- click code view, and the toolbar should be hidden
- if you press esc key, you can be back to the air mode or default mode from code view

#### What are the relevant tickets?
#3570


### Checklist
